lib/utils/report.mjs

```diff
@@ -47,10 +47,9 @@ export default function report(problem) {
 		);
 	}
 
-	const range = node?.rangeBy({ index, endIndex }) ?? {};
 	// If a line is not passed, use the node.rangeBy method to get the
 	// line number that the complaint pertains to
-	const startLine = line ?? range.start?.line;
+	const startLine = line ?? node?.rangeBy({ index, endIndex })?.start?.line;
```

Comment:
If `rangeBy` returns an object then `start` is always present hence one `?` could be dropped.
i.e. `node?.rangeBy({ index, endIndex })?.start.line`;

We could also drop the first `?` but let's not do that in that PR.
i.e. `node` is not optional